app/views/shared/_sidebar.html.erb

```diff
@@ -11,7 +11,14 @@
     <span>
       <%= image_tag 'tribo-logo-light.svg', class: 'sidebar-logo' %>
     </span>
-    <span class="sidebar-item"><button id="modal-register" class="btn text-left">Regístrate / Mi cuenta</button></span> 
+    <% @market_places.each do |market_place| %>
```

Comment:
Try to avoid using `instance var` in partial views. It's preferred to pass the local variables to a partial 😄 

Suggested reading: https://andycroll.com/ruby/dont-use-instance-variables-in-partials/